### ISSUE TITLE
AI (Internal): Current Activity is null for event = 'System.Net.Http.HttpRequestOut.Stop'

### ISSUE THREAD

**Issue by scott-lin** [microsoft/ApplicationInsights-node.js#473]
This trace message is popping up all over our AppInsights instance. I don't know what is means, or what could be the cause. I'm happy to provide any details that can help debug.

**Trace message**
 ```
AI (Internal): Current Activity is null for event = 'System.Net.Http.HttpRequestOut.Stop'
```

**Details for one of the trace items pictured below**
* **AppName**: devautorest-functions
* **Timestamp**: 2018-12-20T01:42:53.5863399Z
* **cloud_RoleInstance**: RD0004FFB2E828
* **sdkVersion**: dotnet:2.7.2-23439
* **itemId**: 99ed4fb1-03f8-11e9-9fa0-35d94a8a1598

**App context**
The node application is running inside of an Azure Function on a Consumption plan. Multiple instances may be running at a time, however, I'm not privy to how function host/process state is created, shared and destroyed. Hope these details help jog ideas.

![image](https://user-images.githubusercontent.com/31644762/50259017-36838c00-03b7-11e9-8a07-905e289911d9.png)

**Comment by OsvaldoRosado** [comment]
@lmolkova do you have context for what would cause this?

These look like messages from the .NET SDK that instruments the underlying Functions runtime rather than the Node.js SDK (the Node SDK doesn’t report traces with “AI (Internal)”)

**Comment by lmolkova** [comment]
This is a known issue https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/1009.
It does not indicate any real issue, just creates noise.

It's fixed, but stable version is not released yet and is not picked up by Azure Functions. We expect to address this early next year.

The workaround for this issue is very complicated and depends on some other active issues. 

@scott-lin how disruptive it is for you? could you tolerate this messages for now?

**Comment by scott-lin** [comment]
Yes, we can tolerate the noise. Thanks for confirming the fix is on the way. Is there a way we can be notified when the fix is deployed?
